api/desecapi/views/authenticated_actions.py

```diff
@@ -196,8 +196,7 @@ def post(self, request, *args, **kwargs):
         return Response(
             {
                 "detail": f"Success! Welcome to deSEC. {self.authenticated_action.token.owner.email} will "
-                "manage your domains for you. Should you want to access your account directly, you can do "
-                "so at any time. (Remember to first set a password for your account.)"
+                "manage your domains for you. Set a password any time to access your account directly."
```

Comment:
"Reset the password to your account at any time to access your account directly."

Reply:
done